bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -1190,6 +1190,18 @@ impl<'a> CircuitInputStateRef<'a> {
                 CallContextField::IsSuccess,
                 0u64.into(),
             );
+
+            //even call.rw_counter_end_of_reversion is zero for now, it will set in
+            //set_value_ops_call_context_rwc_eor later
```

Comment:
Nitpick
```suggestion
            // Even call.rw_counter_end_of_reversion is zero for now, it will set in
            // set_value_ops_call_context_rwc_eor later
```

Reply:
updated in [9206fe2](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1030/commits/9206fe2b5c3d7aad9869992c514b6660ca19a44b) thanks!